src/models/Comment.js

```diff
@@ -0,0 +1,38 @@
+import Sequelize from 'sequelize';
+import db from './db'
+
+const Comment = db.define('comment', {
+  qno: {
```

Comment:
Dont store it like this use relations http://docs.sequelizejs.com/manual/tutorial/associations.html#belongsto

Reply:
Sorry http://docs.sequelizejs.com/manual/tutorial/associations.html#one-to-many-associations